etc/picongpu/taurus-tud/V100_picongpu.profile.example

```diff
@@ -76,7 +76,34 @@ export CXXFLAGS="-Dlinux"
 export TBG_SUBMIT="sbatch"
 export TBG_TPLFILE="etc/picongpu/taurus-tud/V100.tpl"
 
-alias getNode='srun -p ml --gres=gpu:6 -n 1 --mem=0 --cpus-per-task=44 --pty -t 2:00:00 bash'
+# allocate an interactive shell for two hours
+#   getNode 2  # allocates 2 interactive nodes (default: 1)
+function getNode() {
+    if [ -z "$1" ] ; then
+        numNodes=1
+    else
+        numNodes=$1
+    fi
+    export OMP_NUM_THREADS=7
+    srun --time=2:00:00 --nodes=$numNodes --ntasks=$((6 * $numNodes)) --ntasks-per-node=7 --cpus-per-task=7 --mem=0 --exclusive --gres=gpu:6 -p ml --pty bash
```

Comment:
Why do you choose `OMP_NUM_THREADS=7` and `--ntasks-per-node=7`? I think `--ntasks-per-node=6` should be used for 6 gpus / node.

Reply:
I think `OMP_NUM_THREADS=7` is because there are 44 cores per 6 GPUs so 7 is the integer part of 44 / 6. However ntasks should probably 6, not 7, indeed.

Reply:
Yes `--ntasks-per-node` should be `6`. See [here](https://github.com/ComputationalRadiationPhysics/picongpu/blob/fad264596fa8410c6bb1a2ac2d5d627f43153859/etc/picongpu/taurus-tud/V100.tpl#L56)

Reply:
Yes will fix it